history-of-quakers/modernized/12-chapter-10.adoc

```diff
@@ -1127,10 +1243,13 @@ I do hereby require you accordingly to release, and set the said prisoner, Georg
 at liberty: for which this shall be your warrant and discharge.
 Given under my hand the 25th day of October, in the year of our Lord God, 1600.
 
-THOMAS MALLET.
+[.signed-section-signature]
+Thomas Mallet.
 
 To Sir John Lenthal, knight, marshal of the King`'s Bench, or his deputy.
```

Comment:
Here's the other one that I marked [sscc] in the original.

Reply:
cool, let's make this one a [sscc] as well 👍